deployments/helm/nvidia-device-plugin/templates/_helpers.tpl

```diff
@@ -217,3 +217,15 @@ Pod annotations for the plugin and GFD
 {{- end -}}
 {{- toYaml $annotations }}
 {{- end -}}
+
+
+{{/*
+Collection of typed options for the device plugin.
+
+We convert this to JSON so that it can be included and converted to an object using fromJson.
+*/}}
+{{- define "nvidia-device-plugin.options" -}}
+{{- $options := dict "" "" -}}
+{{- $_ := set $options "hasConfigMap" ( eq ( (include "nvidia-device-plugin.hasConfigMap" . ) | trim ) "true" ) -}}
+{{- mustToJson $options -}}
```

Comment:
```suggestion
{{- mustToJson $options -}}
{{- fromJson $options -}}
```

Reply:
I don't think that works as expected. The point is that `options` here must be a JSON string. The reason being that when we include this template it is **always** rendered as a string. This is why we use `fromJson` at the call site to construct a typed object there.

Reply:
ahh yes, makes sense. So part of the workaround is to pipe the output of the `include` call to `fromJson` at the point of invocation.

Unfortunate that we have to resort to this workaround, but I see there's no other way around it. I will resolve the comment